db-schema/postgres/Author.js

```diff
@@ -15,7 +15,7 @@ const Author = sequelize.define(
       allowNull: false,
     },
     biography: {
-      type: DataTypes.STRING,
+      type: DataTypes.TEXT,
```

Comment:
What's the difference between string and text?

Reply:
String is varchar, cant be long. Text is postgres type text, used for long texts